sale_coupon_multi_gift/models/sale_coupon_multi_gift.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class SaleCouponGift(models.Model):
+    _name = "sale.coupon.multi.gift"
```

Comment:
Why not being in the same file than reward one being a o2m child? And the name can also be `sale.coupon.reward.product_line` or similar

Reply:
Changed